src/Arcus.Observability.Telemetry.Core/Logging/RequestLogEntry.cs

```diff
@@ -28,10 +28,53 @@ public class RequestLogEntry
         ///     Thrown when the <paramref name="statusCode"/>'s status code is outside the 100-599 inclusively,
         ///     the <paramref name="duration"/> is a negative time range.
         /// </exception>
+        public RequestLogEntry(
```

Comment:
Can't we call our other constructor to reduce duplication?

Reply:
We should maybe assign a default operation name here when calling the other constructor

Reply:
Changed it so we call the other constructor and assing a default operation, see https://github.com/arcus-azure/arcus.observability/pull/229/commits/8804c4bea20c511e20149bd502a02fe8d8a65640